src/embezzler.ts

```diff
@@ -366,19 +437,20 @@ export const embezzlerSources = [
       get("beGregariousCharges") > 0
         ? 1
         : 0,
-    (options: EmbezzlerFightOptions) => {
+    (options: EmbezzlerFightRunOptions) => {
       adventureMacro(
         $location`The Dire Warren`,
-        Macro.if_($monster`fluffy bunny`, Macro.item($item`human musk`)).step(
-          options.macro ?? embezzlerMacro()
-        )
+        Macro.if_($monster`fluffy bunny`, Macro.item($item`human musk`)).step(options.macro)
       );
     },
-    [
-      new Requirement([], {
-        forceEquip: $items`miniature crystal ball`.filter((item) => have(item)),
-      }),
-    ]
+    {
+      requirements: [
```

Comment:
at the risk of starting a debate that gets me thrown out of every garbo party ever run, i would recommend one of two things here. effectively, "Be Gregarious" and "Final Be Gregarious" are exactly the same EmbezzlerFight. the only difference is that if we have the crystal ball on the 3rd fight, we want to equip the crystal ball. given the lack of differentiation, i would recommend either:
- making them one EmbezzlerFight and adding a condition on the requirement to only equip the orb if there's just 1 greg left
- keeping them separate and adding a note to both that if any changes are made to one, they should be made to the other to try and stop errors from mismatched behavior on the two aligned embezzlerfights

Reply:
I think merging them is fine so long as we now have the tools to do so. Historically requirement was static. If it's a function now, it's very easy to add.

If it's _not_ a function, we can still merge them, but it's spicier. We'd need to equip orb in the run() function, which means that we're getting a suboptimal deal out of our maximizer string--familiar weight is nonlinear wrt meat drop, so that makes it bad.

Another issue is just that Orb is a disaster. I have no solution to this disaster, I just want to hang that albatross on someone else's neck as we discuss this.